themes/default/content/registry/packages/ionoscloud/installation-configuration.md

```diff
@@ -0,0 +1,48 @@
+---
+# WARNING: this file was fetched from https://raw.githubusercontent.com/ionos-cloud/pulumi-ionoscloud/v0.2.1/docs/installation-configuration.md
+# Do not edit by hand unless you're certain you know what you are doing!
+title: Ionoscloud Installation & Configuration
+meta_desc: Information on how to install the Ionoscloud provider.
+layout: installation
+---
+
+## Installation
+
+The Pulumi `Ionoscloud` provider is available as a package in most Pulumi languages:
+
+* JavaScript/TypeScript: [`@ovhcloud/pulumi-ovh`](https://www.npmjs.com/package/@ionos-cloud/sdk-pulumi)
```

Comment:
The text of this link should probably be `@ionos-cloud/sdk-pulumi`

Reply:
@rmocanu-ionos just checking in, do you want to continue with with adding the provider to the registry?

Reply:
Hey, yes, we want to. Sorry, I was under the impression that you need to do something to run the workflow for this pr or something. Should I make a new tag with this change?

Reply:
Yes, please release a new version of the provider with this fix and then I’ll take it from here!

Reply:
I made a new release (v0.2.2) with the change, should I regenerate the docs and add the changes to this pr?

Reply:
No that's fine, I'll need to create a separate PR for verification